spec/facades/new_authorization_request/api_entreprise_facade_spec.rb

```diff
@@ -3,7 +3,7 @@
 
   let(:authorization_definition) { instance_double(AuthorizationDefinition, editors: [editor]) }
   let(:an_already_integrated_editor) { an_object_having_attributes(already_integrated: ['api_entreprise']) }
-  let(:editor) { Editor.all.first }
+  let(:editor) { ServiceProvider.all.first }
```

Comment:
```suggestion
  let(:editor) { ServiceProvider.editors.first }
```

related à mon commentaire plus haut